The URL from which XML test data is fetched is dead.  Redirect the test
to fetch the XML data from the MacPorts guide.

--- test/easy.c.orig	2024-02-21 23:39:18
+++ test/easy.c	2026-09-30 17:35:11
@@ -41,7 +41,7 @@
   check(data, err);
 
   fprintf(stdout, "New data from url... ");
-  data = nxmle_new_data_from_url("http://www.autistici.org/bakunin/index.php",
+  data = nxmle_new_data_from_url("https://raw.githubusercontent.com/macports/macports-guide/refs/heads/master/guide/resources/base.xsl",
                                  &err);
   check(data, err);
 
